expo/tests/gbe/test_bid_class.py

```diff
@@ -73,24 +79,6 @@ def test_bid_class_no_personae(self):
         nt.assert_true(expected_string in response.content)
         nt.assert_equal(response.status_code, 200)
 
-    def test_class_bid_post_with_submit(self):
```

Comment:
this is a duplicate of the function below it.  There's a function by the same name that gets called later and does the real work, this is just superfluous code likely to add confusion.